apps/condo/pages/phone/[number]/index.tsx

```diff
@@ -10,6 +10,7 @@ import { Col, Row, Typography } from 'antd'
 import { CarouselRef } from 'antd/es/carousel'
 import { Gutter } from 'antd/es/grid/row'
 import { EllipsisConfig } from 'antd/es/typography/Base'
+import { isEmpty } from 'lodash'
```

Comment:
Can you please import a specific module? We agreed to use the import of individual modules on the front end `import isEmpty from 'lodash/get'`